pkg/utils/docker.go

```diff
@@ -0,0 +1,40 @@
+package utils
+
+import (
+	"fmt"
+
+	"github.com/Sirupsen/logrus"
+
+	"github.com/kubernetes-incubator/kompose/pkg/kobject"
+)
+
+func DockerBuildImage(name string, service kobject.ServiceConfig, composeFileDir string) string {
+	image := name
+	if service.Image != "" {
+		image = service.Image
+	}
+
+	cmd := NewCommand(fmt.Sprintf("docker build -t %s %s", image, service.Build))
```

Comment:
Also, I think fmt.Springf would be a badddddddddd idea. Someone could plug in `|| rm -rf /` in here as the image name..